test/test_common.c

```diff
@@ -169,6 +172,24 @@ void parse_cmd(int argc, char **argv, test_params *params)
             params->test_resolve_peers = 1;
         } else if( 0 == strcmp(argv[i], "--test-error") ){
             params->test_error = 1;
+        } else if(0 == strcmp(argv[i], "--test-replace") ) {
+            i++;
+            if (NULL != argv[i] && (*argv[i] != '-')) {
+                params->key_replace = strdup(argv[i]);
+                if (0 != parce_replace(params->key_replace, 0, NULL)) {
```

Comment:
@karasevb this function wasn't correctly renamed, thus Mellanox jenkins is failing at compilation stage.